server/utils/lostBedUtils.ts

```diff
@@ -0,0 +1,18 @@
+import type { TemporaryAccommodationLostBed } from '@approved-premises/api'
+
+export const allStatuses: Array<{ name: string; id: TemporaryAccommodationLostBed['status']; tagClass: string }> = [
```

Comment:
I think right now this isn't used, and the data here gets duplicated in `lostBedInfo.ts`. I'd rather `lostBedStatusHtml` work the same as `formatStatus` elsewhere, and this just take the status ID string, then lookup human-readable string/colour

Reply:
generally agree, although the problem with changing lostBedStatusHtml is that it's an unusual case - on the view void booking page we want to render 'Active' or 'Cancelled' to the user, whereas on the view bedspace page, if it's an active void booking we want to render 'Void' to the user

Reply:
I've followed the pattern but added an optional parameter and if/else for the view bedspace screen case

Reply:
Is this two functions then? One that does a similar mapping from status ID to HTML as elsewhere. Then a function in `viewUtils.ts` for when we just want to turn some string and some colour into HTML, and isn't void-specific?